index.html

```diff
@@ -0,0 +1,2110 @@
+
+              <div class="card-wrapper d-flex mb-4">
+                <img class="side-news-img me-3" alt="image" src="asset/images/world-little-2.webp" data-holder-rendered="true" />
+                <div class="card-body card-body-test d-block p-0 m-0 ml-2">
+                  <h3 class="stories-card-title">
+                    <a href="/trump-biden-super-predator-debate-1541565">
+                      Trump Cuts Kwame Kilpatrick's Jail Time for Detroit
+                      Corruption by 20 Years</a>
+                  </h3>
+                </div>
+              </div>
+
+              <div class="card-wrapper d-flex mb-4">
+                <img class="side-news-img me-3" alt="image" src="asset/images/world-little-3.webp" data-holder-rendered="true" />
+                <div class="card-body card-body-test d-block p-0 m-0 ml-2">
+                  <h3 class="stories-card-title">
+                    <a href="/trump-biden-super-predator-debate-1541565">
+                      Video of Police Dog Arlo Going Home After Shooting
+                      Viewed 41 Million Times</a>
+                  </h3>
+                </div>
+              </div>
+            </div>
+            <!-- END OF side news wrapper-->
+          </div>
+          <!-- END OF ROW CONTAINER-->
+        </div>
+        <!-- END OF Newspapers sections Newsweek Expert Forum-->
+
+        <!-- START OF Newspapers sections  Subscribe / Newsletter -->
+        <div class="newp-sect-subs-newsletter container-fluid">
+          <div class="row px-3 mb-2 pt-3 border-top">
+            <div class="subscribe-main-wrapper d-block col-12 col-lg-7 container-fluid">
+              <div class="in-magazine-title text-danger pb-3">Subscribe</div>
+              <div class="subscribe-wrapper d-flex flex-column flex-md-row justify-content-center align-items-center container-fluid">
+                <div id="img-container" class="subscribe-images-wrapper position-relative mb-3 ms-2">
+                  <img id="img-curr" class="subscribe-img" src="asset/images/subscribe-prev.jpg" alt="" />
+                  <img id="img-prev" class="subscribe-img position-absolute" src="asset/images/subscribe-curr.jpg" alt="" />
+                </div>
+                <div class="subscribe-text d-flex flex-column justify-content-center align-items-center mt-1 me-2">
+                  <h4 class="in-magazine-first-card-title d-flex flex-wrap justify-content-center text-center">
+                    <strong>In-depth Stories You Don't See Elsewhere About Topics
+                      You Want to Read!</strong>
+                  </h4>
+                  <p class="text-center">
+                    All Access + Weekly Delivery. <br />
+                    Choose A Membership That's Perfect for You!
+                  </p>
+                  <button type="button" class="btn btn-danger">
+                    SUBSCRIBE
+                  </button>
+                </div>
+              </div>
```

Comment:
- [ ] The subscribe version should have a grey background, check the original site and the screenshot from it below:
![subscribe_and_newsletter_desktop](https://user-images.githubusercontent.com/16193212/106256363-f16aee80-61f9-11eb-8ec9-6f3cb466c487.png)